docker/test/integration/steps/steps.py

```diff
@@ -248,6 +262,27 @@ def step_impl(context, file_name, content, path):
     context.test.add_test_data(path, content, file_name)
 
 
+@given("a Funnel with the name \"{funnel_name}\" is set up in the flow")
+def step_impl(context, funnel_name):
+    funnel = Funnel()
+    funnel.set_name(funnel_name)
+    context.test.add_node(funnel)
+
+
+@given("in the flow the Funnel with the name \"{source_name}\" is connected to the {destination_name}")
```

Comment:
"in the flow" is implicit from the context, I don't think we need it explicitly written
```suggestion
@given("a Funnel with the name \"{funnel_name}\" is set up")
def step_impl(context, funnel_name):
    funnel = Funnel()
    funnel.set_name(funnel_name)
    context.test.add_node(funnel)


@given("the Funnel with the name \"{source_name}\" is connected to the {destination_name}")
```

Reply:
Updated in eb396ae43b5dae3ec5c23f19725f5ac6d380bf78